beerocks/bcl/source/beerocks_os_utils.cpp

```diff
@@ -171,7 +171,8 @@ bool os_utils::touch_pid_file(std::string file_path)
 
 int os_utils::redirect_console_std(std::string log_file_name)
 {
-    int fd_log_file_std = open(log_file_name.c_str(), O_CREAT | O_APPEND | O_RDWR, S_IWUSR | S_IRUSR);
+    int fd_log_file_std =
+        open(log_file_name.c_str(), O_CREAT | O_APPEND | O_RDWR, S_IWUSR | S_IRUSR);
```

Comment:
I prefer whitespace-only changes to be in a separate commit, to make review easier...

Reply:
Basically, for easier code review I agree - our normal process is to add commits on top of each other but eventually squash them. I will make sure to follow this rule in future commits (we should have some document that describes the code review process).